### DISCUSSION TITLE
单点运行出现的问题

### DISCUSSION THREAD

**Discussion by Dylan-leo1** [NCAR/noahmp#165] [Q&A] [answered]
![Uploading image.png…]()
wrfinput_flnm: '../HRLDAS_forcing/run/examples/single_point/hrldas_setup_single_point.nc'
forrtl: severe (24): end-of-file during read, unit 15, file /home/dylan/hrldas-master/hrldas/run/NoahmpTable.TBL, line 2, position 0
Image              PC                Routine            Line        Source             
hrldas.exe         00000000006897C6  for__io_return        Unknown  Unknown
hrldas.exe         00000000006BD965  for_read_seq_nml      Unknown  Unknown
hrldas.exe         00000000005A6505  Unknown               Unknown  Unknown
hrldas.exe         000000000065860E  Unknown               Unknown  Unknown
hrldas.exe         000000000040A7CA  Unknown               Unknown  Unknown
hrldas.exe         000000000040A75E  Unknown               Unknown  Unknown
libc-2.17.so       00007F3D5690C555  __libc_start_main     Unknown  Unknown
hrldas.exe         000000000040A669  Unknown               Unknown  Unknown
不知道为什么会这样，求解答。

**Comment by cenlinhe** [comment]
It may be due to you set up the wrong path for input files and/or run directory. Please double check.
Did you modify the NoahmpTable.TBL file? Please double check you have this file under your run directory.

**Reply by Dylan-leo1** [reply, marked as answer]
Thank you very much, because the NoahmpTable.TBL is not complete, now it has basically run successfully.
